package.json

```diff
@@ -62,6 +62,8 @@
     "zod": "^3.16.0"
   },
   "dependencies": {
-    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#1.3.0"
+    "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#1.3.0",
+    "json-schema-to-typescript": "^10.1.5",
+    "ts-to-zod": "^1.10.0"
```

Comment:
Just double checking that these need to be in the dependencies vs the dev-dependencies.

Reply:
Fixed, thanks for catching that!